contexts/credentials/v2

```diff
@@ -43,17 +44,20 @@
     },
     "cnf": {
       "@id": "https://www.iana.org/assignments/jwt#cnf",
-      "@context": {
-        "@protected": true,
-        "kid": {
-          "@id": "https://www.iana.org/assignments/jwt#kid",
-          "@type": "@id"
-        },
-        "jwk": {
-          "@id": "https://www.iana.org/assignments/jwt#jwk",
-          "@type": "@json"
+      "@context": [
+        null,
+        {
+          "@protected": true,
```

Comment:
I assume this is just preventing someone from changing term definitions... what would be a use case for doing that?

Why is protected required for this object?

The other part that this section does seems to fix the way the graph looks by default, which seems great.

Reply:
> I assume this is just preventing someone from changing term definitions... what would be a use case for doing that?

Yes. The use case is making sure that people don't re-define the JWT terms to use some other URL value (that doesn't point to the IANA registries).

> Why is protected required for this object?

For the reason above.

> The other part that this section does seems to fix the way the graph looks by default, which seems great.

Yep.